sklearn/exceptions.py

```diff
@@ -37,6 +39,21 @@ class NotFittedError(ValueError, AttributeError):
     """
 
 
+class InvalidParameterError(ValueError):
+    """ TODO
+    """
+
+    def __init__(self, param_name):
+        self.param_name = param_name
+
+
+class SignatureError(RuntimeError):
```

Comment:
I don't see why this should be public

Reply:
You're right. I didn't think about that. Anyway, this will be gone as per the comment above.